recipe/meta.yaml

```diff
@@ -91,6 +91,13 @@ requirements:
     - python
     - pytorch                                          # [cuda_compiler_version in (undefined, 'None')]
     - tqdm
+    {% if cuda_major >= 12 %}
+    - cuda-compiler
+    - cuda-cudart-dev
+    - libcusparse-dev
+    - libcublas-dev
+    - libcusolver-dev
```

Comment:
These cuda libraries are slightly different to the ones listed under `host` above. Just want to confirm that this is the correct list, i.e. there are no extra ones needed for JiT compilation?